tests/Application/config/packages/fos_rest.yaml

```diff
@@ -5,6 +5,8 @@ fos_rest:
             json: true
             xml:  true
         empty_content: 204
+    zone:
+        - { path: '^/api/.*' }
```

Comment:
What's this? :)

Reply:
This is the fix for that weird issue. See:

Fixed: Unable to find template "" (looked into: ...) issue …
3d824cc
See: https://github.com/Sylius/Sylius-Standard/pull/389/files